internal/rbd/rbd_util.go

```diff
@@ -72,6 +72,10 @@ const (
 	// thick provisioned or thin provisioned.
 	thickProvisionMetaData = "true"
 	thinProvisionMetaData  = "false"
+
+	// these are the migration label key and value for params in volume context.
```

Comment:
` params in volume context` to `parameters in volume context`

Reply:
done.. thanks..